lib/policyEvaluator/evaluator.js

```diff
@@ -0,0 +1,257 @@
+'use strict'; // eslint-disable-line strict
+
+const substituteVariables = require('./utils/variables.js');
+const handleWildcards = require('./utils/wildcards.js').handleWildcards;
+const conditions = require('./utils/conditions.js');
+const findConditionKey = conditions.findConditionKey;
+const convertConditionOperator = conditions.convertConditionOperator;
+const checkArnMatch = require('./utils/checkArnMatch.js');
+
+const evaluators = {};
+
+
+/**
+ * Check whether resource in policy statement applies to request resource
+ * @param {object} requestContext - info about request
+ * @param {string | [string]} statementResource - Resource(s) impacted
+ * by policy statement
+ * @return {boolean} true if applicable, false if not
+ */
+function isResourceApplicable(requestContext, statementResource) {
+    const resource = requestContext.resource;
+    if (!Array.isArray(statementResource)) {
+        // eslint-disable-next-line no-param-reassign
+        statementResource = [statementResource];
+    }
+    // ARN format:
+    // arn:partition:service:region:namespace:relative-id
+    const requestResourceArr = resource.split(':');
+    // Pull just the relative id because there is no restriction that it
+    // does not contain ":"
+    const requestRelativeId = requestResourceArr.slice(5).join(':');
```

Comment:
I think a better way to compute the relative id is to take the length of the 5 first part and then do a substring, because here, we create an array without the first 5 five part, and then we create a string with the remaining parts :)